src/menu/ide_menu.rs

```diff
@@ -559,8 +559,13 @@ impl IdeMenu {
         if use_ansi_coloring {
             if index == self.index() {
                 format!(
-                    "{}{}{}{}{}{}{}",
+                    "{}{}{}{}{}{}{}{}",
                     vertical_border,
+                    suggestion
+                        .style
+                        .unwrap_or(self.color.text_style)
+                        .reverse()
```

Comment:
Seems like this has the same problem that I had with rsteube's PR, it's hard coding `reverse()` which would ignore any setting you have in nushell. it should also be implemented in nushell since it's just adding a style.

Shouldn't it be implemented more like this? https://github.com/nushell/reedline/pull/720/files

Reply:
thanks for catching that!
i just made #725

Reply:
thank you!